.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,11 @@
-Description:
-<!--
-Replace this template with your PR description.
-Please remember to keep in mind the security levels outlined in
-[CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md) and to
-include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title
-
-* **no**: None
-* **low**: Low chance of potential impact to, or exposure of patient data
-* **moderate**: Moderate chance of potential impact to, or exposure of patient data
-* **severe**: Severe chance of potential impact to, or exposure of patient data
-
-Please also:
-
-* Get thumbs from reviewer(s)
-* Verify all tests go green, including CI tests
--->
-
+**Description**
 
 ---
 **PR checklist**
 
-- [ ] This PR meets the Acceptance Criteria in the JIRA story
-- [ ] The JIRA story has been moved to Dev Review
-- [ ] This PR includes appropriate unit tests
-- [ ] I have added explanatory comments where the logic is not obvious
-- [ ] I have run and tested this change locally, and my testing process is described here
+- [ ] If this PR is associated with a Jira issue, I have included an issue number tag in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
+- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
+- [ ] I have manually tested this change
+- [ ] This PR includes appropriate automated tests, and I have documented any behavior that cannot be tested with code
 - [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
-- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
 - [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
-- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
-- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
-  * AoU UI
-  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
-  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
-  * Cron tasks - for Offline*Controllers
-  * SumoLogic - for EgressAlert 
```

Comment:
Is this something that happens often enough to warrant checking every single PR?

This seems like a strange place to document our API consumers. Most of the breakages this item is trying to detect would be picked up by our type system or our tests. We rarely modify SumoLogic.

Reply:
I remember that we did think this was worthwhile when adding it.  May be worth revisiting now, of course.

Perf tests -> no longer exist, so we can remove that.
RDR Export is a cron task so we can group with them -> we can catch problems here by looking at cron error logs.  (we should also be more diligent about this)

OK, seems we can probably remove this checkbox.

But I would like to keep the deployment-safety check.

Reply:
I re-added the deployment-safety check.